pennylane/devices/tests/test_properties.py

```diff
@@ -123,10 +108,22 @@ def test_model_is_defined_valid_and_correct(self, device_kwargs):
         assert "model" in cap
         assert cap["model"] in ["qubit", "cv"]
 
-        qnode = qml.QNode(qfunc_no_input, dev)
+        if cap["model"] == "qubit":
+
+            @qml.qnode(dev)
+            def circuit():
+                qml.PauliX(wires=0)
+                return qml.expval(qml.PauliZ(wires=0))
+
+        else:
+
+            @qml.qnode(dev)
+            def circuit():
+                qml.Displacement(1.0, 1.2345, wires=0)
+                return qml.expval(qml.X(wires=0))
```

Comment:
We skip the tests if the device is CV based, so we should be able to get away with the qubit definition only.